psi4/driver/procrouting/proc.py

```diff
@@ -1371,6 +1371,42 @@ def scf_helper(name, post_scf=True, **kwargs):
         core.print_out("""  PCM does not make use of molecular symmetry: """
                        """further calculations in C1 point group.\n""")
         use_c1 = True
+    
```

Comment:
Since a 3-layer QM/MM/PCM is currently not possible, we need to check that both options are not set at the same time and abort if that's the case.

Reply:
Resolved in c3cd34779.